modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -997,23 +1021,31 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       throw new InvalidAddressError(`invalid address: ${address}`);
     }
 
-    if ((_.isUndefined(chain) && _.isUndefined(index)) || !(_.isFinite(chain) && _.isFinite(index))) {
-      throw new InvalidAddressDerivationPropertyError(
-        `address validation failure: invalid chain (${chain}) or index (${index})`
-      );
-    }
+    let expectedAddress: AddressDetails;
+    if (params.coinSpecific && 'descriptor' in params.coinSpecific) {
+      expectedAddress = this.generateAddress({
+        descriptor: params.coinSpecific.descriptor,
```

Comment:
we only submit the name of the descriptor (the full descriptor itself is defined on the wallet)